srcpkgs/python-peewee/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'python-peewee'
 pkgname=python-peewee
-version=3.5.1
+version=3.5.2
```

Comment:
Why don't you use the latest [release](https://github.com/coleifer/peewee/releases) 3.6.3 ?

Reply:
Because I opened this PR before 3.6 was released. Updated to 3.6.3 now.